Source/Atomic/Network/MasterServerClient.cpp

```diff
@@ -0,0 +1,556 @@
+//
+// Created by Keith Johnston on 4/16/16.
+//
```

Comment:
There needs to be a license block on new source files, for example: https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/AtomicTool/AtomicTool.cpp#L1

You can copy that and apply your name also add yourself to the AUTHORS.md :)  https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/AUTHORS.md